tests/unit/src/main/scala/tests/TestingServer.scala

```diff
@@ -1352,6 +1352,10 @@ final class TestingServer(
     )
     server.references(params).asScala.map { r =>
       r.asScala
+        .sortBy { l =>
```

Comment:
Ahh, ha much easier than I thought. Awesome. Out of curiosity, how come this comes back in different orders? I thought this always came in the correct order?

Reply:
I think it might be sorted just by the lines somewhere, which is normally not an issue since we just highlight the correct place.